Source/WarpX.H

```diff
@@ -1491,6 +1492,12 @@ private:
      */
     void PSATDVayDeposition ();
 
+    // TODO Add Doxygen string
+    void PSATDComputeCurrentPartialSums ();
+
+    // TODO Add Doxygen string
+    void PSATDSubtractCurrentPartialSumsAvg ();
```

Comment:
These must be `public` member functions, otherwise you cannot use host-device lambdas in its implementation
```
/home/runner/work/WarpX/WarpX/Source/FieldSolver/WarpXPushFieldsEM.cpp(611): error #3196-D: The enclosing parent function ("PSATDSubtractCurrentPartialSumsAvg") for an extended __device__ lambda cannot have private or protected access within its class
```